README.md

```diff
@@ -1,6 +1,44 @@
 [![Pursuit Logo](https://avatars1.githubusercontent.com/u/5825944?s=200&v=4)](https://pursuit.org)
 
-# [Assignment Title]
+# Pursuit Core Web Props Lab
+
+![goFundMe](./goFundMe.png)
+
+Included in this repo is an [html file](./reference.html) that renders a static version in HTML.
+Build a React app modeled after the image above.
+
+> Most of your class names and HTML elements can be copied from the reference file.
+
+It should use each of the following components should be included under `src/Components` as a separate `.js` file, like `src/Components/TopBar.js`.
```

Comment:
I think there are missing words between `components` and `should`

Reply:
_words_